index.html

```diff
@@ -61,6 +61,14 @@ <h1 class="display-3">Bikes for Refugees</h1>
 							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
 						</div>
 					</div>
+					<div class="container row justify-content-between">
```

Comment:
You cannot use a container and a row on the same level. The order must be: 
1) `container` (grandparent)
2) `row` (parent)
3) `col-...` (children) **note: must add up to 12**
each level must be the direct child of the other.

Reply:
If you follow this, structure, your 3 boxes will align with the rest of that block.